kernels/volk/volk_32f_s32f_convert_32i.h

```diff
@@ -237,18 +237,20 @@ static inline void volk_32f_s32f_convert_32i_generic(int32_t* outputVector,
 {
     int32_t* outputVectorPtr = outputVector;
     const float* inputVectorPtr = inputVector;
-    unsigned int number = 0;
-    float min_val = INT_MIN;
-    float max_val = INT_MAX;
-    float r;
+    const float min_val = (float)INT_MIN;
+    const float max_val = (float)INT_MAX;
 
-    for (number = 0; number < num_points; number++) {
+    for (unsigned int number = 0; number < num_points; number++) {
+        float r;
+        int s;
         r = *inputVectorPtr++ * scalar;
```

Comment:
One last comment. I suggest to merge line 244 with 246.

```suggestion
        const float r = *inputVectorPtr++ * scalar;
```
I'd declare it `const` as well. This helps others understand the code better.